### PR TITLE
Disable core dump generation for openj9 load tests

### PR DESCRIPTION
**Pull request by Mesbah-Alam**
Disable core dump generation for openj9 load tests
Signed-off-by: Mesbah_Alam@ca.ibm.com <Mesbah_Alam@ca.ibm.com>

### CODE REVIEW

**Comment by Mesbah-Alam** [comment]
Tested with: https://ci.adoptopenjdk.net/view/Test_grinder/job/Grinder/784/tapResults/

**Comment by Mesbah-Alam** [comment]
Reviewer : @pshipton

File: openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/test/sc/SCSoftmxTestUtil.java [hunk 1]
```diff
@@ -137,7 +137,7 @@ public static LoadTestProcessDefinition getMiniMixLoadTestOptions(LoadTestProces
 			.addProjectToClasspath("openjdk.test.classloading")  // For sharedclasses-mix inventory
 			.addProjectToClasspath("openjdk.test.nio")           // For sharedclasses-mix inventory
 			.addProjectToClasspath("openj9.test.daa")         // For sharedclasses-mix inventory
-		   	.generateCoreDumpAtFirstLoadTestFailure(false)
+		   	.generateCoreDumpAtFirstLoadTestFailure(true)
```
Comment (pshipton) [inline openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/test/sc/SCSoftmxTestUtil.java]: Why is this one changed from false to true?

**Comment by Mesbah-Alam** [comment]
Background: After speaking with Peter Shipton, decided to turn off core file generation of these tests. The cores/failures don't cause the tests to fail, so nobody would look at them. Moreover, these core files continue to make builds fail on AIX due to lack of disk space issue.

File: openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/test/sc/SCSoftmxTestUtil.java (hunk 1 above)
Comment (Mesbah-Alam) [reply openj9.test.sharedClasses/src/test.sharedClasses/net/openj9/test/sc/SCSoftmxTestUtil.java]: That was mistake. Fixed.

**Comment by Mesbah-Alam** [comment]
@pshipton - could you please merge this PR?

**Pull request merged**